### PR TITLE
add `referential_constraints` under "information-schema"

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
added a document `referential_constraints` under "information-schema"

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/6139
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-referential-constraints.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: REFERENTIAL_CONSTRAINTS
+summary: 了解 information_schema 表 `REFERENTIAL_CONSTRAINTS`。
+---
+
+# REFERENTIAL_CONSTRAINTS
+
+`REFERENTIAL_CONSTRAINTS`表提供了有关表之间`FOREIGN KEY`关系的信息。请注意，目前 TiDB 不对`FOREIGN KEY`进行约束，也不执行`ON DELETE CASCADE`等操作。
```
Comment (TomShawn) [inline information-schema/information-schema-referential-constraints.md:8]:
```suggestion
`REFERENTIAL_CONSTRAINTS` 表提供表之间 `FOREIGN KEY` 关系的信息。请注意，目前 TiDB 不对 `FOREIGN KEY` 进行约束，也不执行 `ON DELETE CASCADE` 等操作。
```

**Comment by TomShawn** [comment]
https://github.com/pingcap/docs/pull/6139/files#diff-af901fcf2c80c7a2e8e277df009f5454dbdd631c9298f1cfd46d9197c3baaefdR35 这一行未对齐。

**Comment by TomShawn** [comment]
@en-jin19 Have you missed this comment?

> https://github.com/pingcap/docs/pull/6139/files#diff-af901fcf2c80c7a2e8e277df009f5454dbdd631c9298f1cfd46d9197c3baaefdR35 这一行未对齐。

**Comment by en-jin19** [comment]
> @en-jin19 Have you missed this comment?
> 
> > https://github.com/pingcap/docs/pull/6139/files#diff-af901fcf2c80c7a2e8e277df009f5454dbdd631c9298f1cfd46d9197c3baaefdR35 这一行未对齐。

Sorry, I missed it indeed. I have added it just now, please kindly take a look.

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**